cmd/server/main.go

```diff
@@ -4,12 +4,57 @@ import (
 	"todoapp/internal/adapter/http/rest"
 	"todoapp/internal/adapter/memdb"
 	"todoapp/internal/common/config"
+	"todoapp/internal/entity"
+	"todoapp/internal/usecase/repo"
 )
 
 func main() {
 	userRepo := memdb.NewUserRepo()
+	todoRepo := memdb.NewTodoRepo()
 
-	s := rest.NewServer(userRepo)
+	initUserData(userRepo)
```

Comment:
Delete these two init methods immediately, these testing support methods are not allowed on shared branched